helm/single-content/resources/agentstore.cfg

```diff
@@ -0,0 +1,164 @@
+// patching config file
```

Comment:
I think if this file is going here then it ought to be named qms-agentstore.cfg, though we should think whether to keep these configs in this chart or with the chart deploying that agentstore.

Reply:
https://github.com/opentext-idol/idol-containers-toolkit/pull/27/commits/bf18e6e89e83dfef5073f46f104f3e8dbe6dddc5